### PR TITLE
T3

### PR DESCRIPTION
**Pull request by john-harmony**
Update T3 with latest master branch for OSTN P2 prep

### CODE REVIEW

File: shard/shard_state.go [hunk 1]
```diff
@@ -382,23 +385,52 @@ func (c *Committee) DeepCopy() Committee {
...
+				if err := c.Slots[j].BlsPublicKey.ToLibBLSPublicKey(
+					committerKey,
+				); err != nil {
+					return nil, err
+				}
+				slice[j] = committerKey
+			}
+			// Only made once
+			go func() {
+				time.Sleep(25 * time.Minute)
```
Comment (LeoHChen) [inline shard/shard_state.go]: why 25 minutes hardcoded?

Comment (rlan35) [reply shard/shard_state.go]: it's just a reasonable time to keep the cached result.

Comment (LeoHChen) [reply shard/shard_state.go]: opened a ticket to keep track of this issue which has to be fixed later.

Comment (LeoHChen) [reply shard/shard_state.go]: https://github.com/harmony-one/harmony/issues/2559

**Pull request merged**